torch/_meta_registrations.py

```diff
@@ -2012,6 +2012,76 @@ def meta_scatter_reduce__two(self, dim, index, src, reduce, include_self=True):
     return self
 
 
+def multiply_integers(vs):
+    r = 1
+    for v in vs:
+        r *= v
+    return r
```

Comment:
I swear we already have a copy of this function lol

Reply:
True, I found one in `torch/jit/_shape_functions.py`